settings/settings.php

```diff
@@ -44,7 +44,7 @@ function render_custom_ui() : void {
 		return;
 	}
 
-	echo '<div class="notice notice-info notice-alt"><p>This interface and functionality is currently a <em>beta release</em>, while it is currently operational it\'s not yet officially released. Please report any issues you encounter to <a href="https://github.com/WordPress/wporg-two-factor/issues">WordPress/wporg-two-factor on GitHub</a>.</p></div>';
+	echo '<div class="notice notice-info notice-alt"><p>This interface and functionality is currently a <em>beta release</em>, while it is currently operational it\'s not yet officially released. Please report any issues you encounter to <a href="https://github.com/WordPress/wporg-two-factor/issues" data-content="notice-info-link">WordPress/wporg-two-factor on GitHub</a>.</p></div>';
```

Comment:
What do you like about using `data-content` over `class`? `class` seems more conventional to me, but I'm curious if there's a benefit to `data-content` that I'm not aware of.

Reply:
I believe the use of `data-content` comes from trying to find a solution to the issue `Screen reader stops reading paragraph when it reaches anchor tag`, but in the end, I found this actually is an accessible-friedly way: It gives the user the opportunity to follow the links when the screenreader reaches them and waits before it proceeds. `class` should be used here. Thanks for pointing out 🙏

Reply:
Fixed in https://github.com/WordPress/wporg-two-factor/pull/125/commits/1ef8591a093b3110d5dbb54d274bc55d1b003122